misc/config_tools/schema/types.xsd

```diff
@@ -70,18 +89,21 @@
 severity and intent:
 
 - 1 (LOG_FATAL) system is unusable
-- 2 (LOG_ACRN) hypervisor failure
-- 3 (LOG_ERROR) error conditions
-- 4 (LOG_WARNING) warning conditions
-- 5 (LOG_INFO) informational
-- 6 (LOG_DEBUG) debug-level messages
+- 2 (LOG_ERROR) error conditions
+- 3 (LOG_WARNING) warning conditions
+- 4 (LOG_INFO) informational
+- 5 (LOG_DEBUG) debug-level messages
 
 A lower value has a higher severity. Log messages with a
 higher value (lower severity) are discarded.</xs:documentation>
   </xs:annotation>
-  <xs:restriction base="xs:integer">
-    <xs:minInclusive value="0" />
-    <xs:maxInclusive value="7" />
+  <xs:restriction base="xs:string">
+    <xs:enumeration value="0: None" />
```

Comment:
This is to ensure that the enumeration values shown in the UI are pretty-printed, because previously I think the enumerations shown in the configurator (which uses jsonschema) must be the same as those defined in the schema.

However, I was just told today that there is a possibility to define "pretty" labels to those enumerations in the configurator while keeping the underlying values as magic as they are today. Thus I'll refine the patches accordingly.